app/models/manageiq/providers/ibm_cloud/power_virtual_servers/manager_mixin.rb

```diff
@@ -37,17 +37,8 @@ def connect(options = {})
   end
 
   def api_endpoint_url(location)
-    api_endpoint_overrides = ::Settings.ems.ems_ibm_cloud_power_virtual_servers.api_endpoint_overrides
-
-    if api_endpoint_overrides.key?(location.to_sym)
-      url = api_endpoint_overrides[location.to_sym]
-    else
-      region = location.sub(/-\d$/, '')
-      region = region.sub(/\d\d$/, '')
-      url = "#{region}.power-iaas.cloud.ibm.com"
-    end
-
-    url
+    region = region.sub(/-*\d+$/, '')   # e.g. 'syd04' -> 'syd'
```

Comment:
Need to call the sub method on the `location` string, and assign it to `region`:
```suggestion
    region = location.sub(/-*\d+$/, '')   # e.g. 'syd04' -> 'syd'; 'eu-de-2' -> 'eu-de'
```